tracing/providers/jaeger/src/main/java/io/helidon/tracing/providers/jaeger/JaegerTracerBuilder.java

```diff
@@ -335,13 +351,61 @@ public JaegerTracerBuilder trustedCertificates(io.helidon.common.configurable.Re
         return this;
     }
 
+    /**
+     * Span Processor type used.
+     *
+     * @param spanProcessorType to use
+     * @return updated builder
+     */
+    @ConfiguredOption(key = "span-processor-type", value = "batch")
+    public JaegerTracerBuilder spanProcessorType(SpanProcessorType spanProcessorType) {
+        this.spanProcessorType = spanProcessorType;
+        return this;
+    }
+
+    /**
+     * Schedule Delay of exporter requests.
+     *
+     * @param scheduleDelayMillis timeout to use
+     * @return updated builder
+     */
+    @ConfiguredOption(key = "schedule-delay", value = "5000")
```

Comment:
The value is not correct, it should be `PT5S` (if it is 5 seconds)

Reply:
fixed